### DISCUSSION TITLE
i can't use generateStaticParams to generate Arabic letters path for dynamic segments

### DISCUSSION THREAD

**Discussion by HishamBasha** [vercel/next.js#52603] [App Router]
i don't exactly know what is wrong with Arabic letters ..    i used export const dynamicParams = false; to show me 404 if no dynamic route was generated for my [slug] segment, then for Arabic characters i got 404, but when i visit a path with non-Arabic characters no 404 is shown

**Comment by AmirrezaRezaeii** [comment]
Same here...
non-english dynamic segments in dynamic routes occurs not-found 404 after build time
any solution ?
